examples/integration_test.py

```diff
@@ -50,14 +51,26 @@
 # on each connect with this set to True.
 api_version_request = True
 
-# global variable to be set by stats_cb call back function
+# global variable to be set by stats_cb callback function
 good_stats_cb_result = False
 
+# global variable to be incremented by throttle_cb callback function
+throttled_requests = 0
+
 
 def error_cb(err):
     print('Error: %s' % err)
 
 
+def throttle_cb(t_report):
+    global throttled_requests
+    throttled_requests += 1
+
+    print('Request to broker %s[id=%d] throttled for %d ms' % (t_report.get('broker_name'),
```

Comment:
Use proper type here and verify that object attributes are set and have the expected type.

Reply:
This is handled by Py_BuildValue() in confluent_kafka.c. Now that the arguments are passed as a tuple which is immutable and accessible only though the ThrottleEvent object I'm not sure if this is still needed.